packages/service-library/src/data-service/page-scan-run-report-service.ts

```diff
@@ -18,22 +17,26 @@ export class PageScanRunReportService {
     public async saveSarifReport(fileId: string, content: string): Promise<void> {
         await this.blobStorageClient.uploadBlobContent(
             PageScanRunReportService.blobContainerName,
-            this.getBlobFilePath(fileId, PageScanRunReportService.sarifFileExtension),
+            this.getBlobFilePath(fileId, this.getBlobSarifFileName(fileId)),
             content,
         );
     }
 
     public async readSarifReport(fileId: string): Promise<BlobContentDownloadResponse> {
         return this.blobStorageClient.getBlobContent(
             PageScanRunReportService.blobContainerName,
-            this.getBlobFilePath(fileId, PageScanRunReportService.sarifFileExtension),
+            this.getBlobFilePath(fileId, this.getBlobSarifFileName(fileId)),
         );
     }
 
-    private getBlobFilePath(fileId: string, extension: string): string {
+    private getBlobFilePath(fileId: string, fileName: string): string {
         const fileCreatedTime = this.guidGenerator.getGuidTimestamp(fileId);
 
         return `${fileCreatedTime.getUTCFullYear()}/${fileCreatedTime.getUTCMonth() +
-            1}/${fileCreatedTime.getUTCDate()}/${fileCreatedTime.getUTCHours()}/${fileId}${extension}`;
+            1}/${fileCreatedTime.getUTCDate()}/${fileCreatedTime.getUTCHours()}/${fileName}`;
+    }
+
+    private getBlobSarifFileName(fileId: string): string {
+        return `${fileId}_sarif.json`;
```

Comment:
the extension has to be sarif & not json

Reply:
wan't fix, there is no such well known extension as sarif, json is more commonly supported and will reflect file formatting to be used by viewers